src/sources/aws_kinesis_firehose/errors.rs

```diff
@@ -19,6 +19,7 @@ pub enum RequestError {
     #[snafu(display("Could not parse incoming request {}: {}", request_id, source))]
     Parse {
         source: serde_json::error::Error,
+        access_key: String,
```

Comment:
It doesn't seem like you're using this field - but if it's confidential enough to be in the "secret" portion of the event, would we even want to log it in plaintext?

Reply:
I'm not sure in what condition I wrote this code, you are right – just removed it. The only thing that I remember, that I was actually trying to pass `access_key` down to the `firehose` from warp 🫠